app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,854 @@
+	<string name="currently_worth">obecnie warta</string>
+	<string name="located_in">Zlokalizowana w</string>
+	<string name="wishlist">Lista Życzeń</string>
+	<string name="wishlist_comment">Koemntarz do Listy Życzeń</string>
+	<string name="title_trade">Handel</string>
+	<string name="trade_condition">Stan gry</string>
+	<string name="want_parts_list">Lista chcianych częśći</string>
+	<string name="has_parts_list">Lista posiadanych części</string>
+	<string name="msg_colors_generated">Wygenerowano kolory!</string>
+	<string name="drawer_open">Otwórz menu nawigacyjne</string>
+	<string name="drawer_close">Zamknij menu nawigacyjne</string>
+	<string name="message_custom_player_order">Chcesz zostawić obecnie pozycje startowe czy je wyczyścić?</string>
+	<string name="play_stat_distinct_games">Różne gry</string>
+	<string name="play_stat_quarters">Quarters</string>
+	<string name="play_stat_quarter">Quarter</string>
+	<string name="play_stat_half_dollar">Half-dollar</string>
+	<string name="play_stat_half_dollars">Half-dollars</string>
+	<string name="play_stat_dollar">Dollar</string>
+	<string name="play_stat_dollars">Dollars</string>
+	<string name="play_stat_dimes">Dimes</string>
+	<string name="play_stat_dime">Dime</string>
+	<string name="play_stat_penny">Penny</string>
+	<string name="play_stat_win_percentage">%1$,d/%2$,d (%3$,d%%)</string>
+	<string name="percentage">%,d%%</string>
+	<string name="title_win_skill">Biegłość</string>
+	<string name="winnable">Do wygrania</string>
+	<string name="all">Wszystkie</string>
+	<string name="play_stat_average_play_time">Średni czas rozgrywki</string>
+	<string name="play_stat_average_play_time_faster">Szybciej niż przewidywano</string>
+	<string name="play_stat_average_play_time_slower">Wolniej niż przewidywano</string>
+	<string name="play_stat_average_play_time_per_player">Średni czas gry na gracza</string>
+	<string name="play_stat_nickels">Nickels</string>
+	<string name="play_stat_nickel">Nickel</string>
+	<string name="play_stat_first_play">Pierwsza rozgrywka</string>
+	<string name="play_stat_last_play">Ostatnia rozgrywka</string>
+	<string name="play_stat_top_100">100 najczęściej granych gier</string>
+	<string name="play_stat_game_h_index">Wskaźnik H-index dla gier</string>
+	<string name="play_stat_game_h_index_info">Zagrałeś w min. %1$,d gier %1$,d lub więcej razy. Aby zwiększyć tę liczbę potrzebujesz min. %2$,d rozgrywek.\n\nhttps://boardgamegeek.com/thread/953084</string>
+	<string name="play_stat_game_h_index_offset_in">Ta gra wlicza się do Twojego wskaźnika H-index.</string>
+	<string name="play_stat_game_h_index_offset_out">rozgrywek poza Twoim wskaźnikiem H-index.</string>
+	<string name="play_stat_player_h_index">Wskaźnik H-index dla graczy</string>
+	<string name="play_stat_player_h_index_info">Zagrałeś przeciwko min. %1$,d graczom %1$,d lub więcej razy. Aby zwiększyć tę liczbę potrzebujesz min. %2$,d rozgrywek.\n\nhttps://boardgamegeek.com/thread/1448479</string>
```

Comment:
It sounds good now but will POSSIBLY sound better as plural. (Only) SIMILAR to `<plurals name="person_game_h_index_info">` at the end of the file. 
BUT, unlike the line 605 above… IF `%1$,d` is inserted only when greater than 1… then this line is great and it doesn't need to be a plural. I need to know if `%1$,d` can =1.